tests/framework/ngf.go

```diff
@@ -110,6 +111,9 @@ func InstallNGF(cfg InstallationConfig, extraArgs ...string) ([]byte, error) {
 
 	if cfg.ServiceType != "" {
 		args = append(args, formatValueSet("service.type", cfg.ServiceType)...)
+		if cfg.ServiceType == "LoadBalancer" && cfg.IsGKEInternalLB {
+			args = append(args, formatValueSet(`service.annotations.networking\.gke\.io\/load-balancer-type`, "Internal")...)
```

Comment:
I wonder why escaping is needed before .gke and .io, but not before .annotations or .networking?

Reply:
Because `annotations` here corresponds to the Yaml object, but all the remaining dots are part of the annotation itself. The dot before `networking` indicates to helm a new `break`. The formatted result is like this:
```
<...>
kind: Service
metadata:
  <...>
  annotations:
    networking.gke.io/load-balancer-type: "Internal"
<...>
```

Reply:
thanks for clarification!!